packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -642,23 +638,48 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 	/**
 	 * Closes the connection and clears inbound & outbound queues.
 	 *
-	 * @param doDispose - should the DeltaManager treat this close call as a dispose?
-	 * Differences between close and dispose:
-	 * - dispose will emit "disposed" event while close emits "closed"
-	 * - dispose will remove all listeners
-	 * - dispose can be called after closure, but not vis versa
+	 * Differences from dispose:
+	 * - close will switch connection to readonly
```

Comment:
The connection is disposed.  Then it calls back to DM to notify the world of readonly.

```suggestion
	 * - close will trigger readonly notification
```